src/gl/directx11/shaders/fenceVertexShader.hlsl

```diff
@@ -25,15 +25,22 @@ cbuffer u_EveryFrame : register(b0)
 {
   float4 u_bottomColour;
   float4 u_topColour;
+  float4 u_worldUp; //can represent both vertical and horizontal orientations
```

Comment:
Why not calculate expand vector on CPU and avoid all the work in the shader?

Reply:
Oh this comment shouldn't be here. The expand vectors are still created cpu side. We have a vertex-specific expand vector and a global world up. The shader just switches between the two depending on whether we want to render fences or paths.